.github/workflows/ci.yml

```diff
@@ -1546,6 +1550,9 @@ jobs:
             xtensa-intel_tgl_adsp_zephyr-elf)
               PLATFORM_ARGS+="-p intel_adsp_cavs25 "
               ;;
+            xtensa-mt8195_adsp_zephyr-elf)
+              PLATFORM_ARGS+="-p mt8195_adsp "
```

Comment:
Do we have a PR introducing this? We need to have this ready somehow and merge it or put it in the sdk branch to get this test to build, otherwise CI will fail.

Reply:
PR for the board is at https://github.com/zephyrproject-rtos/zephyr/pull/61197 but I wasn't pushing it as there are still some review comments and IMHO it makes more sense to wait for the call0 work to land first as then it can launch with working userspace (it's got an MPU).

Isn't that a chicken and egg problem?  I assumed we needed the SDK patch before the board could merge too?  Maybe I should remove these lines and merge an update later?

Reply:
it is a chicken and egg problem, but when we test the SDK in CI, we pull this from a branch (topic-sdk-dev), so we can put this PR for building purposes and while we wait for it to get merged in zephyr main.